keyboards/centromere/rules.mk

```
@@ -0,0 +1,41 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+F_CPU = 8000000
+F_USB = $(F_CPU)
+
+# Build Options
+#   comment out to disable the options.
+#
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
```

```suggestion
BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration
```
Yuo may want to make this `lite` instead: https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite